podman-next/Containerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/fedora/fedora-coreos:stable
+
+# Note: This needs to be updated to the next Fedora release manually. The repo
+# config itself is release agnotic but the link will be broken once the F36
+# buildroot is removed.
+ENV PODMAN_COPR="https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/repo/fedora-36/rhcontainerbot-podman-next-fedora-36.repo"
```

Comment:
Fine as is, but I'd consider instead committing both of these repo files to git, and using a `COPY` command.  Particuarly if you commit the GPG key to git, it's a bit more secure.

(Alternatively of course, in the future we should probably add `dnf copr enable`)

Reply:
Eh, we can do further tweaks in followups, no reason to hold this.